app/components/tag_list_form_component.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# Create tag list for exhibit general/create form
+class TagListFormComponent < ViewComponent::Base
```

Comment:
component should be namespaced in `module Spotlight` and moved to `app/components/spotlight`